tests/test_frame_selection.py

```diff
@@ -84,7 +85,29 @@ def test_tt_rms():
     assert len(pruned_dataset) == 5
 
     # removes 2 frames
-    pruned_dataset = frame_select(default_dataset, tt_thres=2.5)
+    pruned_dataset = frame_select(default_dataset, tt_rms_thres=2.5)
+    assert len(pruned_dataset) != len(default_dataset)
+    assert len(pruned_dataset) == 3
+
+def test_tt_bias():
```

Comment:
doesn't this have to be added to the bottom of the file in `if __name__ == "__main__"`?